SU-HSDO/suhumsci
# fix(shs-5068): adding max width to sidebar form

# READY FOR REVIEW

## Summary
Ensures that an exposed filter in the three column sidebar layout does not bleed outside the column width. 

## Urgency
medium

## Steps to Test
For reproducing the bug in Traditional/Colorful sites using the Courses views:

- Visit /admin/structure/views/view/hs_default_courses/edit/list , enable the exposed forms block and save (see step 2 above)
- Visit /courses  and click on "Layout" in the local actions tab
- Go to the left sidebar, click on "Add Block". Search for "exposed" and add the "Default Courses: All List Exposed form" block
- Save the layout. You should see the filters on the left sidebar
- Ensure that the form fields do not go beyond the column (like in this screenshot):

<img width="574" alt="Screenshot 2023-09-19 at 11 52 54" src="https://github.com/SU-HSDO/suhumsci/assets/2343957/b73cab80-3124-4523-a384-cc872e7444c3">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
